docs/source/components.rst

```diff
@@ -240,6 +261,12 @@ Cross field validation is made possible by overriding ``Form.clean()``:
             return data
 
 
+If you use a form's cleaned data to create a new model then model validation
+will also take place. It can be customized by passing a list of validators to a
```

Comment:
This is only true if `ModelForm.is_valid` is called. Passing `form.cleaned_data` to `Model.save` doesn't automatically call any model validation.